tests/test_gui_glfw.py

```diff
@@ -185,6 +186,7 @@ def draw_frame():
     canvas.set_logical_size(300, 200)
     canvas.set_logical_size(400, 300)
     for i in range(5):
+        time.sleep(0.01)
```

Comment:
oh boy, that looks bad

Reply:
Yeah, well, this gives other threads/processes some slack. Apparently it's needed. It may be that `sleep(0)` (which introduces a thread-switch) might already have worked, but I did not try that.